src/tools/cephfs/top/cephfs-top

```diff
@@ -208,7 +224,56 @@ class FSTop(object):
             # If the terminal do not support the visibility
             # requested it will raise an exception
             pass
-        self.run_display()
+        fs_list = self.get_fs_names()
+        self.run_all_display(fs_list)
+
+    def display_menu(self, stdscr, selected_row_idx):
+        stdscr.clear()
+        h, w = stdscr.getmaxyx()
+        title = ['Filesystems', 'Press "q" to see all filesystems!']
```

Comment:
Suggestion: I feel `!` could be removed, just "Press 'q' to see all filesystems" looks better.